### DISCUSSION TITLE
Welcome to OFB_2023 Discussions!

### DISCUSSION THREAD

**Discussion by PBissonDFG** [Data-For-Good-Ottawa/OFB_2023#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Salma-Al-Shehabi** [comment]
Hello,
My name is Salma. I would like to gain practical experience and collaborate in this project.
